modules/nf-core/cellpose/meta.yml

```
@@ -24,7 +24,7 @@ input:
       pattern: "*.{tif,tiff}"
   - model:
       type: file
-      description: Optional input file. Cellpose 2 model trained by user using human-in-the-loop approach.
+      description: Optional input file. Cellpose model trained by user using human-in-the-loop approach.
```

```suggestion
      description: Optional input file. Cellpose 2 model trained by user using human-in-the-loop approach.
```
Custom model training was added with Cellpose 2. I would keep this.